### DISCUSSION TITLE
Padding in p-button

### DISCUSSION THREAD

**Discussion by angeasla** [primefaces/community#28] [PrimeNG] [answered]
Hello. Is there any way to remove the padding inside a p-button. I want the text to be at the far left of the button.

**Comment by SoyDiego** [comment]
If you share your code, will be easier to help you or share a stackblitz please.
you can use `::ng-deep` to access and edit the styles of the elements.

**Comment by yigitfindikli** [comment]
Hi @angeasla!

@SoyDiego is right.

Different approach:

You can use the styleClass property of p-button.

- p-0 (You can use your own classes.) is a class of [PrimeFlex](https://primeflex.org/padding).

`<p-button label="Submit" styleClass="p-0"></p-button>`

https://stackblitz.com/edit/vjkbkd?file=src%2Fapp%2Fdemo%2Fbutton-basic-demo.html

**Reply by angeasla** [reply, marked as answer]
Thank you and @SoyDiego for your immediate responses. I've done it by just adding 'pl-0' in 'styleClass' like this:

```
<p-button styleClass="p-button-text pl-0" label="label"
            (click)="goToSite()"></p-button>
```

**Comment by SoyDiego** [comment]
> Thank you and @SoyDiego for your immediate responses. I've done it by just adding 'pl-0' in 'styleClass' like this:
> 
> ```
> <p-button styleClass="p-button-text pl-0" label="label"
>             (click)="goToSite()"></p-button>
> ```

Ah yes, If you are using primeflex, you can do that.
My reply is if you are not using that and you need to access inside the button to apply the style :)
I'm glad you were able to do it.
